modules/ROOT/pages/to-find-info.adoc

```diff
@@ -30,54 +13,137 @@ Exchange Search looks for terms in:
 * The content of the asset portal pages
 * xref:to-find-info.adoc#searching-api-specifications[Properties defined in RAML and OAS API specifications]
 
-You can combine search terms with the filters listed in All Types.
+In Exchange you can enter search text to search for assets, click *All Types* to set a filter for specific asset types, or combine search text with the filters listed in the type menu.
+
+<<index.adoc#content-types,Content Types>> lists the different types of asset.
+
+You can also search by clicking category or tag elements. This creates a filtered search you can use to find similar assets.
+
+The original Exchange search experience is available in Anypoint Platform Private Cloud Edition, MuleSoft Government Cloud, the US cloud, and the EU cloud. The new Exchange search experience is available in the US cloud and the EU cloud.
+
+New Exchange search experience:
 
-You can also search by clicking Category or Tag elements. This creates a filtered search you can use to find similar assets.
+image::search.png[New Exchange search experience]
 
-In Exchange Private Cloud Edition and GovCloud, searches apply only to the Anypoint Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
+In the original Exchange search experience, searches apply only to the Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
 
-In Exchange Cloud, you can select which of the organizations visible to you should be included in each search.
+In the new Exchange search experience, you can select which of the organizations visible to you should be included in each search.
```

Comment:
Can we add a sentence here explaining that by default (i.e. if the user does not select any specific organization in the dropdown filter) the search will be executed against the whole customer organization (master organization plus it's business groups) AND against Mulesoft's public assets?

Reply:
That's a great idea! Added.